### PR TITLE
Accepting an Issue creates a Reversion object

### PR DESCRIPTION
**Pull request by fragalcer**
Issues resolved: #

Sanity checks:
- [ ] Have tests been run locally?
- [ ] Does this change any javascript? If so, have you checked that the UI works as expected?
- [ ] If there are data model changes, are relevant data standards referenced - in code?
- [ ] If there are deviations from any specified standards, are the reasons for changes documented?
- [ ] If there are UI changes have you documented whats changed so the help documentation can be updated?

### CODE REVIEW

File: python/aristotle-metadata-registry/aristotle_mdr/contrib/issues/urls.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
-from django.conf.urls import url
+from django.urls import path
```
Comment (LegoStormtroopr) [inline python/aristotle-metadata-registry/aristotle_mdr/contrib/issues/urls.py]: Can we roll this back?
More in slack.

File: python/aristotle-metadata-registry/aristotle_mdr/contrib/issues/models.py [hunk 2]
```diff
@@ -61,18 +62,20 @@ def can_view(self, user):
     def can_alter_open(self, user):
         return self.can_edit(user) or self.item.can_edit(user)
 
-    def apply(self):
-        """Apply proposed changes to an item"""
+    def apply(self, user):
```
Comment (LegoStormtroopr) [inline python/aristotle-metadata-registry/aristotle_mdr/contrib/issues/models.py]: Is there a risk that user could be an AnonymousUser?
If so, will that break reversion?
@harrywhite4 @fragalcer

**Comment by LegoStormtroopr** [comment]
LGTM, just one question above.

File: python/aristotle-metadata-registry/aristotle_mdr/contrib/issues/models.py (hunk 2 above)
Comment (s-i-l-k-e) [reply python/aristotle-metadata-registry/aristotle_mdr/contrib/issues/models.py]: The _IssueApproveView_ requires that you have edit permission on the item before changes are applied. :+1:

**Pull request merged**